homeassistant/components/media_player/services.yaml

```
@@ -64,6 +64,13 @@ volume_set:
           min: 0
           max: 1
           step: 0.01
+    volume_step:
+      required: false
+      selector:
+        number:
+          min: 0.001
+          max: 0.1
+          step: 0.01
```

I think the scale is off here... Volume is 0-1 (i.e. percentage), so this should use the same scale. The minimum being 1% and the max being 100%:
```suggestion
          min: 0.01
          max: 1.0
          step: 0.01
```